src/libFLAC/include/private/cpu.h

```diff
@@ -154,6 +154,7 @@ typedef enum {
 	FLAC__CPUINFO_TYPE_IA32,
 	FLAC__CPUINFO_TYPE_X86_64,
 	FLAC__CPUINFO_TYPE_PPC,
+	FLAC__CPUINFO_TYPE_ARM,
```

Comment:
Revisited this today, and I think FLAC__CPUINFO_TYPE_ARM64 is a more accurate name